templates/github/.github/workflows/update_ci.yml.j2

```diff
@@ -44,6 +44,11 @@ jobs:
           ref: '{{ branch }}'
           fetch-depth: 0
 
+      - name: Checkout update_ci.sh from main
+        working-directory: {{ plugin_name }}
+        run: |
+          git checkout main -- .github/workflows/scripts/update_ci.sh
```

Comment:
This is needed because the branch to which the plugin template is being applied to is checked out before the update_ci.sh script is run. We are now in a situation where there is a bug in the update_ci.sh on the 3.28 branch and even though we fixed it on the main branch, the script is executed from the 3.28 branch.

Reply:
I am thinking that instead of checking out the file from main branch, we should just make a copy of it on the main branch and then use that copy of it.

Reply:
I now understand what you are saying. Yes, let's move the script to the plugin_template repository and call it from there!

Reply:
@gerrod3 I implemented the changes suggested by @mdellweg! Thanks for the suggestion @mdellweg